product_docs/docs/pglogical/3.7/configuration.mdx

```diff
@@ -0,0 +1,301 @@
+---
+navTitle: PostgreSQL Configuration
+title: PostgreSQL settings which affect pglogical
+originalFilePath: configuration.md
+
+---
+
+Several PostgreSQL configuration options may need adjusting for pglogical
+to work.
+
+PostgreSQL must be configured for logical replication:
+
+```
+wal_level = 'logical'
+```
+
+The pglogical library need to be loaded at server start, so the parameter
+`shared_preload_libraries` must contain pglogical, e.g.:
+
+```
+shared_preload_libraries = 'pglogical'
+```
+
+As pglogical uses additional worker processes to maintain state and apply
+the replicated changes, enough worker process slots need to be present:
+
+```
+max_worker_processes = 10
+```
+
+The formula for computing the correct value of `max_worker_processes` is:
+one for instance + one per database on the provider (upstream), one for instance
+
+-   one per database + two per subscription on the subscriber (downstream).
```

Comment:
this rendered incorrectly. it should be one paragraph without a bullet in the middle: 

The formula for computing the correct value of max_worker_processes is: one for instance + one per database on the provider (upstream), one for instance + one per database + two per subscription on the subscriber (downstream).

Reply:
Fixing this in situ for now; smells like a parser bug, but I'll have to investigate another time.